charts/meilisearch/values.yaml

```diff
@@ -113,7 +113,7 @@ ingress:
 
 persistence:
   # -- Enable persistence using PVC
-  enabled: false
+  enabled: true
```

Comment:
I don't know why I noticed that earlier...

You changed this to `true` but the PR description only mentions that the idea is to provide the PVC for the plain k8s version. Why this have to change then?

Reply:
Initially I thought that plain k8s version is separate from helm charts, but when I saw that sync-manifest command generates k8s yaml files bases on charts values, so I updated this value to be `true`. I think it's a sensible default - having persistent storage for the index of your search cluster.